client/state/notices/middleware.js

```diff
@@ -74,6 +77,24 @@ import { onJetpackModuleActivationActionMessage } from './jetpack-modules';
  * Handlers
  */
 
+export const onDeleteInvitesFailure = ( dispatch, action, getState ) => {
+	action.inviteIds.map( inviteId => {
+		const invite = getInviteForSite( getState(), action.siteId, inviteId );
+		dispatch(
+			errorNotice(
+				translate( 'An error occurred while deleting the invite for %s.', {
+					args: truncate( invite.user.email, { length: 20 } ),
+				} )
+			)
+		);
+	} );
+};
+
+export const onDeleteInvitesSuccess = ( dispatch, { inviteIds } ) =>
+	dispatch(
+		successNotice( translate( 'Invite deleted.', 'Invites deleted.', { count: inviteIds.length } ) )
```

Comment:
Hi! I've found a possible matching string that has already been translated **58** times:
`translate( 'Invite Deleted.' )` *ES Score: 13*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/invites/revoke-clear)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).